database/migrations/2023_01_13_142805_update_coupons_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::table('coupons', static function (Blueprint $table) {
+            $table->boolean('is_distributed')->after('id')->default(false);
```

Comment:
It seems more logical to make this just the last column?

Reply:
To me it seemed logical to put it next to `integration_id` because they belong together to calculate the status.
But regardless it doesn't change the working of the application so putting new columns at the end of a table might also be preferable.